docs/runtime/go-metrics.md

```diff
@@ -0,0 +1,173 @@
+<!-- endsemconv -->
+
+<!-- semconv metric.go.memory.limit(full) -->
+<!-- endsemconv -->
+
+### Metric: `go.memory.allocated`
+
+This metric is [recommended][MetricRecommended].
+This metric is obtained from Go's [`runtime/metrics`][RuntimeMetrics] package using `/gc/heap/allocs:bytes`.
+
+<!-- semconv metric.go.memory.allocated(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `go.memory.allocated` | Counter | `By` | Memory allocated to the heap by the application. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+<!-- semconv metric.go.memory.allocated(full) -->
+<!-- endsemconv -->
+
+### Metric: `go.memory.allocations`
+
+This metric is [recommended][MetricRecommended].
+This metric is obtained from Go's [`runtime/metrics`][RuntimeMetrics] package using `/gc/heap/allocs:objects`.
+
+<!-- semconv metric.go.memory.allocations(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `go.memory.allocations` | Counter | `{allocation}` | Count of allocations to the heap by the application. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+<!-- semconv metric.go.memory.allocations(full) -->
+<!-- endsemconv -->
+
+## Go Garbage Collection
+
+**Description:** Go metrics captured under the namespace `go.memory.gc.*`
+
+### Metric: `go.memory.gc.target`
+
+This metric is [recommended][MetricRecommended].
+This metric is obtained from Go's [`runtime/metrics`][RuntimeMetrics] package using `/gc/heap/goal:bytes`.
+
+<!-- semconv metric.go.memory.gc.target(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `go.memory.gc.target` | UpDownCounter | `By` | Heap size target for the end of the GC cycle. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+<!-- semconv metric.go.memory.gc.target(full) -->
+<!-- endsemconv -->
+
+### Metric: `go.memory.gc.user_target`
+
+This metric is [recommended][MetricRecommended].
+This metric is obtained from Go's [`runtime/metrics`][RuntimeMetrics] package using `/gc/gogc:percent`.
+
+<!-- semconv metric.go.memory.gc.user_target(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `go.memory.gc.user_target` | UpDownCounter | `1` | Heap size target ratio for the end of the GC cycle, as configured by the user. [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+**[1]:** The value range is [0.0,1.0].
```

Comment:
Why not use the native `[0.0, 100.0]` exported by the runtime package?

Reply:
There are no other % metrics I can find in the semantic conventions.  All others are ratios.  For example, process.cpu.utilization is a ratio, rather than a percentage. It also aligns with the prometheus conventions of using base units.

Reply:
I don't seem to be able to resolve this. Feel free to if you can.